packages/netlify-cms-widget-markdown/src/serializers/__tests__/remarkPlugins.spec.js

```diff
@@ -0,0 +1,291 @@
+import registry from 'netlify-cms-core';
+import visit from 'unist-util-visit';
+
+import { markdownToRemark, remarkToMarkdown } from '..';
+
+describe('registered remark plugins', () => {
+  afterEach(() => {
+    registry._clearRemarkPlugins();
+  });
+
+  function withNetlifyLinks() {
+    return transformer;
+    function transformer(tree) {
+      visit(tree, 'link', function onLink(node) {
+        node.url = 'https://netlify.com';
+      });
+    }
+  }
+
+  it('should use remark transformer plugins when converting mdast to markdown', () => {
+    registry.registerRemarkPlugin(withNetlifyLinks);
+    const result = remarkToMarkdown({
+      type: 'root',
+      children: [
+        {
+          type: 'paragraph',
+          children: [
+            {
+              type: 'text',
+              value: 'Some ',
+            },
+            {
+              type: 'emphasis',
+              children: [
+                {
+                  type: 'text',
+                  value: 'important',
+                },
+              ],
+            },
+            {
+              type: 'text',
+              value: ' text with ',
+            },
+            {
+              type: 'link',
+              title: null,
+              url: 'https://netlify.com',
```

Comment:
```suggestion
              url: 'https://this-value-should-be-replaced.com',
```

as I believe the test passes without registering the plugin

Reply:
ugh, thanks for catching this!